### DISCUSSION TITLE
password reset

### DISCUSSION THREAD

**Discussion by mbarber86** [hassio-addons/addon-tasmoadmin#184] [Q&A] [answered]
I am unable to log into the TasmoAdmin web UI, I suspect I am using the wrong password.  How can I reset the admin password?

**Comment by frenck** [comment, marked as answer]
There is no method to reset a password. Only un-installing and re-installing can provide a new password, but you'll lose your data and settings in the process.

**Reply by LordNex** [reply]
@frenck  So where do you specify the user id and password to use? Even when I reinstall it just loads up and runs and when I go to the web ui it asks for a username and password. What's the default?

I used this a few years back and haven't really needed to make any changes in a while. I've since reinstalled Home Assistant and honestly don't remember how I originally set it up. But I could have sworn that there was a default UID and Pass that you started with and then changed after the initial login. 

Let me know if I need to create this in an issue
